repose-aggregator/components/filters/cors-filter/src/main/scala/org/openrepose/filters/cors/CorsFilter.scala

```diff
@@ -211,7 +211,7 @@ class CorsFilter @Inject()(configurationService: ConfigurationService)
     val maybeForwardedHostUri = Try(maybeForwardedHost
       .map(forwardedHost => new URIBuilder(s"${request.getScheme}://$forwardedHost"))
       .map(uri => uri.setPort(normalizePort(uri.getPort, uri.getScheme)).setHost(normalizeHost(uri.getHost)).build()))
-    lazy val hostUri = new URIBuilder().setScheme(request.getScheme).setHost(normalizeHost(request.getServerName))
+    lazy val hostUri = new URIBuilder("").setScheme(request.getScheme).setHost(normalizeHost(request.getServerName))
```

Comment:
What's this change about? It seems like switching from the empty constructor to the string constructor with an empty string would only change the path of the URI that gets built, but I haven't looked into it too much. If we want to change the path, though, I would suggest calling `.setPath` on the builder instead. Of course, I'm not sure why we are touching this class as part of this PR to begin with...

Reply:
This was a weird problem @dimtruck ran into, and this ended up being the fix. Don't remember what it was related too though.